src/pages/add-kit-item/add-kit-item.ts

```diff
@@ -0,0 +1,120 @@
+import { Component } from '@angular/core';
+import { NavController, NavParams, ModalController, Events } from 'ionic-angular';
+
+import { InventoryData } from '../../providers/inventory-data';
+
+import { Actions, ItemProperties, Messages } from '../../constants';
+import { ItemFilterPage } from '../item-filter/item-filter';
+import { Notifications } from '../../providers/notifications';
+
+@Component({
+  selector: 'page-add-kit-item',
+  templateUrl: 'add-kit-item.html'
+})
+export class AddKitItemPage {
+  itemProperties = ItemProperties;
```

Comment:
It's because the template passes in the item property to `presentModal()` when clicking on the input and it cannot access it without having it as a class property.